.github/workflows/python-package.yml

```diff
@@ -12,11 +12,12 @@ on:
 jobs:
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9", "3.10"]
+        os: ["ubuntu-latest", "windows-latest", "macos-latest"]
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
```

Comment:
remove version 3.6 as it doesn't have dataclasses yet

Reply:
I will do this once my computer gets fixed asap